Source/Scene/Model/MetadataPipelineStage.js

```diff
@@ -151,17 +151,15 @@ function getPropertyAttributeInfo(propertyAttribute, primitive, statistics) {
   const classId = propertyAttribute.class.id;
   const classStatistics = statistics?.classes[classId];
 
-  return Object.entries(propertyAttribute.properties).map(
-    getPropertyAttributePropertyInfo
-  );
+  const propertiesArray = Object.entries(propertyAttribute.properties);
```

Comment:
I'm still using `Object.entries` here, simply because I want to know the length of the final array. 
If I simply looped over the raw object, I wouldn't know the length ahead of time, and would have to use `Array.push`, which seems expensive (repeated re-allocation).